docs/3.0rc/develop/write-tasks/use-task-runners.mdx

```diff
@@ -16,27 +16,25 @@ enables you to run tasks concurrently, or in parallel using a distributed execut
 ### Submitting tasks
 
 Using the `.submit()` method to submit a task also causes the task run to return a 
-[`PrefectFuture`](/3.0rc/api-ref/prefect/futures/#prefect.futures.PrefectFuture), which is an 
+[`PrefectConcurrentFuture`](/3.0rc/api-ref/prefect/futures/#prefect.futures.PrefectConcurrentFuture), which is an 
```

Comment:
I think we may want to keep references to `PrefectFuture` since it's the interface, and the specific class of the returned object will change depending on the task runner. Users generally don't need to be aware of the different `PrefectFuture` implementations.
```suggestion
[`PrefectFuture`](/3.0rc/api-ref/prefect/futures/#prefect.futures.PrefectFuture), which is an 
```

Reply:
Good call, I'll revert!